specification.md

```diff
@@ -204,11 +204,36 @@ implementation of a job management API is a job management library. A
 job management library, through its  API,  is invoked by a client
 application.
 
-Traditionally, job management is implemented on supercomputers by Local
-Resource Managers (LRMs), such as PBS/Torque, SLURM, etc. To a first
-approximation, a job management API is understood as an abstraction layer
-on top of various LRMs.
-
+Traditionally, job management is implemented on supercomputers by Local Resource
+Managers (LRMs), such as PBS/Torque, SLURM, etc. To a first approximation, a job
+management API is understood as an abstraction layer on top of various LRMs.
+Consequently, the scope of the present API is informed by functionality commonly
+found across LRMs.
+
+The main motivation behind the present job management API is the ubiquity with
+which projects meant to simplify the process of doing science on compute
+clusters are forced to implement their own solution. The reason is simple. If
+any tool needs to be portable across multiple clusters (and it rarely makes
+sense to not want such portability), it must access the underlying cluster LRM
+in an abstract way; that is, it must use a job management API. The only stable
+job management API currently available is
+[SAGA](http://radical-cybertools.github.io/) [so, wait a minute, how do we
+justify not pushing SAGA forward?].
```

Comment:
I think I will need to leave others to clarify this.  One argument I heard on the call (but not one I would make, just repeating) is that SAGA, or at least the name, comes with a certain pre-notation / ballast, and that would at least partially justify to distinguish an Exaworks job API from it.

BTW: DRMAA also is an existing, stable API -- just not a widely implemented one (anymore).   As such, we should also motivate why we do not implement DRMAA instead of inventing our own API.